pkgs/development/python-modules/jaraco_collections/default.nix

```diff
@@ -10,6 +10,8 @@ buildPythonPackage rec {
     sha256 = "be570ef4f2e7290b757449395238fa63d70a9255574624e73c5ff9f1ee554721";
   };
 
+  pythonNamespaces = [ "jaraco" ];
```

Comment:
I  think this would fix the error I was seeing, without needing the change to `python-namespaces-hook.sh`, since for these packages the `__pycache__/__init__*` expands to only one file.

There are packages that have multiple `__init__*` files, but I'm not sure they are ceph dependencies.

If they are not, it might be faster to merge the ceph fixes straight to `master`, and only send the `python-namespaces-hook` fix through `staging`.

Reply:
I tried that, and it wasn't expanding.

Reply:
and I was worried about python potentially introducing many .pyc's